fe/fe-core/src/main/java/com/starrocks/sql/ast/AstTraverser.java

```diff
@@ -108,9 +108,6 @@ public R visitSubquery(SubqueryRelation node, C context) {
 
     @Override
     public R visitSetOp(SetOperationRelation node, C context) {
-        if (node.hasWithClause()) {
```

Comment:
why remove it?

Reply:
line 112 is same as line 114. So I remove it. But the line 112 may be `node.getCteRelations().forEach(this::visit)`. Maybe it's a bug, let me check.

Reply:
It's ok to remove it. When traverse the ast, the cteRelation will be visited by the `visitCTE(CTERelation node, C context)`.